packages/server/graphql/mutations/helpers/addTeamMemberToMeetings.ts

```diff
@@ -13,6 +13,10 @@ import createMeetingMembers from './createMeetingMembers'
  */
 
 const setInPhase = (phase: CheckInPhase | UpdatesPhase, newStage: CheckInStage | UpdatesStage) => {
+  const isMemberInMeeting = phase.stages.find(
```

Comment:
ideally, the code would never reach this function if the team member is already in the meeting.
what is the condition where this function would get called when a team member is possible already in the meeting? we should identify that & exit a few functions earlier vs. waiting until this function